doc/build-windows-mingw.md

```diff
@@ -0,0 +1,260 @@
+Building Bitcoin on Native Windows
+==================================
+
+This document describes the installation of the mysys and mingw tools as well as 
+the build dependencies and then finally the compilation of the bitcoin binaries.
+
+NOTE: This script will only prepare a build system and bitcoin in 32 bit.
+
+Prepare your build system
+-------------------------
+
+1.1 Install msys shell:
+http://sourceforge.net/projects/mingw/files/Installer/mingw-get-setup.exe/download
+From MinGW installation manager -> All packages -> MSYS
+mark the following for installation:
+
+msys-base-bin
+msys-autoconf-bin
+msys-automake-bin
+msys-libtool-bin
+
+Make sure no mingw packages are checked for installation or present from a previous install. Only the above msys packages should be installed. Also make sure that msys-gcc and msys-w32api packages are not installed.
+
+then click on Installation -> Apply changes
+
+
+1.2 Install MinGW-builds project toolchain:
+Download http://sourceforge.net/projects/mingw-w64/files/Toolchains%20targetting%20Win32/Personal%20Builds/mingw-builds/4.9.2/threads-posix/dwarf/i686-4.9.2-release-posix-dwarf-rt_v3-rev1.7z/download
+and unpack it to C:\
```

Comment:
We should add a clarification here.  The default extract options on 7zip will extract to `C:\i686-4.9.2-release-posix-dwarf-rt_v3-rev1\mingw32\`, but the instructions don't want the `i686-4.9.2-release-posix-dwarf-rt_v3-rev1` directory.

Using the 7zip UI I did the following:
1. Right-click on the downloaded file and choose "Extract files...".  It doesn't matter where the .7z file is located.
2. In the "Extract to:" path, make sure it reads C:\
3. Uncheck the check box directly below the "Extract to" path.
4. Uncheck the "Eliminate duplication of root folder" checkbox.
5. Extract the files.
This way everything is at `C:\mingw32\` where we want it.

![7zip](https://cloud.githubusercontent.com/assets/6402604/17614231/9a23347a-6032-11e6-9b84-ec726b469eb4.png)